src/style.css

```diff
@@ -0,0 +1,45 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
```

Comment:
You should add separate font-faces for 2 separate font-weight (one for normal, one for bold)